src/content/docs/accounts/accounts-billing/new-relic-one-user-management/user-type.mdx

```diff
@@ -584,7 +574,10 @@ Here are more details about how user type impacts access to applied intelligence
 On January 12, 2022, basic users had some log management capabilities removed. Their current capabilities are described below. 
 </Callout>
 
-Basic users can only view and search the [log management UI](/docs/logs/get-started/get-started-log-management). They don't have access to the more advanced log management features, such as [patterns](/docs/logs/ui-data/find-unusual-logs-log-patterns), [live tail](/docs/logs/troubleshooting/view-log-messages-real-time-live-tail), [parsing](/docs/logs/ui-data/parsing), and [data partitions](/docs/logs/ui-data/data-partitions).  
+Log-related capabilities: 
+* Basic users: can only view and search the [log management UI](/docs/logs/get-started/get-started-log-management). They don't have access to the more advanced log management features described below. 
```

Comment:
@zuluecho9, I'm not a fan of separating subjects from verbs with colons. What do you think about either  of these approaches:

* **Basic users:** View and search the [log management UI](/docs/logs/get-started/get-started-log-management). They don't have access to the more advanced log management features described below.

Or this:

* **Basic users** can only view and search the [log management UI](/docs/logs/get-started/get-started-log-management). They don't have access to the more advanced log management features described below.

If you stick with your current colon pattern, you'd want to lower-case "Have" in the third bullet so it matches the others.